homeassistant/components/victron_gx_mqtt/strings.json

```
@@ -0,0 +1,1306 @@
+        "name": "Platform device reboot"
+      }
+    },
+    "number": {
+      "evcharger_set_current": {
+        "name": "EV charger set current"
+      },
+      "generator_gen_id_cool_down_timer": {
+        "name": "Generator cool down timer"
+      },
+      "generator_gen_id_qh_start_on_soc": {
+        "name": "Generator QH start on SOC"
+      },
+      "generator_gen_id_qh_start_on_voltage": {
+        "name": "Generator QH start on voltage"
+      },
+      "generator_gen_id_qh_stop_on_soc": {
+        "name": "Generator QH stop on SOC"
+      },
+      "generator_gen_id_qh_stop_on_voltage": {
+        "name": "Generator QH stop on voltage"
+      },
+      "generator_gen_id_service_interval": {
+        "name": "Generator service interval"
+      },
+      "generator_gen_id_shut_down_timer": {
+        "name": "Generator shut down timer"
+      },
+      "generator_gen_id_start_on_soc": {
+        "name": "Generator start on SOC"
+      },
+      "generator_gen_id_start_on_soc_timer": {
+        "name": "Generator start on SOC timer"
+      },
+      "generator_gen_id_start_on_temp_timer": {
+        "name": "Generator start on temp timer"
+      },
+      "generator_gen_id_start_on_voltage": {
+        "name": "Generator start on voltage"
+      },
+      "generator_gen_id_start_on_voltage_timer": {
+        "name": "Generator start on voltage timer"
+      },
+      "generator_gen_id_stop_on_soc": {
+        "name": "Generator stop on SOC"
+      },
+      "generator_gen_id_stop_on_soc_timer": {
+        "name": "Generator stop on SOC timer"
+      },
+      "generator_gen_id_stop_on_temp_timer": {
+        "name": "Generator stop on temp timer"
+      },
+      "generator_gen_id_stop_on_voltage": {
+        "name": "Generator stop on voltage"
+      },
+      "generator_gen_id_stop_on_voltage_timer": {
+        "name": "Generator stop on voltage timer"
+      },
+      "generator_gen_id_warm_up_timer": {
+        "name": "Generator warm up timer"
+      },
+      "multi_ess_min_soc_limit": {
+        "name": "ESS Minimum SoC Limit"
+      },
+      "multiplus_assist_current_boost_factor": {
+        "name": "Assist Current Boost Factor"
+      },
+      "switch_output_dimming": {
+        "name": "Switch {output} Dimming"
+      },
+      "system_ac_export_limit": {
+        "name": "AC Export Limit"
+      },
+      "system_ac_input_limit": {
+        "name": "AC Input Limit"
+      },
+      "system_ac_power_set_point": {
+        "name": "AC Power Setpoint"
+      },
+      "system_ess_max_charge_current": {
+        "name": "ESS max charge current"
+      },
+      "system_ess_max_charge_voltage": {
+        "name": "ESS max charge voltage"
+      },
+      "system_ess_max_feed_in_power": {
+        "name": "ESS max feed-in power"
+      },
+      "system_ess_max_inverter_power_limit": {
+        "name": "ESS max inverter power limit"
+      },
+      "system_ess_min_soc_limit": {
+        "name": "ESS min SOC limit"
```

We should stick to one spelling of "SoC" throughout the integration.

```suggestion
        "name": "ESS min SoC limit"
```